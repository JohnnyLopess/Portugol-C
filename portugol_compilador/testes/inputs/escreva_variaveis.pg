programa{
    funcao inicio(){
        inteiro x = 2
        escreva("Iteração: ", x)
    }
}