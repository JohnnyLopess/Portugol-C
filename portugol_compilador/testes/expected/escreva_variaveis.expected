#include <stdio.h>
#include <string.h>

int main() {
    int x = 2;
    printf("Iteração: %d\n", x);
    return 0;
}